app/invocation/invocation_execution_card.tsx

```diff
@@ -184,6 +185,15 @@ export default class ExecutionCardComponent extends React.Component {
     });
   }
 
+  handleSearch(execution: execution_stats.Execution) {
+    if (this.props.model.getIsRBEEnabled()) {
+      window.location.search = "?" + execution.actionDigest?.hash + "/" + execution.actionDigest?.sizeBytes;
+      window.location.hash = "#action";
+      return;
+    }
+    router.navigateToSetup();
```

Comment:
Couple things here:
* Maybe rename "handleSearch" to "handleActionDigestClick"
* No need to check if RBE is enabled, since this execution card can only be shown on remotely executed actions
* query params are usually formatted like `?key1=value1&key2=value2`, so the query string here could be `?actionDigest=HASH/SIZE_BYTES`
* When changing `window.location.search`, I think that causes a full page reload. If you do `router.navigateTo("/invocation/INVOCATION_ID?actionDigest=HASH/SIZE#actions")` then it will set the URL (without reloading) using [pushState](https://developer.mozilla.org/en-US/docs/Web/API/History/pushState) and tells the root React component to re-render with the new URL props, without re-loading the app